### PR TITLE
#989: Page editor add/insert panel updates

### PR DESCRIPTION
**Pull request by BLoe**
This updates some button text and tweaks styling in the add/insert extension panels in page editor.

**Generic Add New Panel:**
image

**Insert Button:**
image

**Insert Panel:**
image

### CODE REVIEW

**Comment by BLoe** [comment]
I'm gonna tweak some of the text here

**Comment by BLoe** [comment]
Ok, should be good to go here unless you want to tweak any of the text any further @twschiller

File: src/devTools/editor/panes/insert/InsertMenuItemPane.tsx [hunk 1]
```diff
@@ -49,9 +54,9 @@ const InsertMenuItemPane: React.FunctionComponent<{ cancel: () => void }> = ({
 
       <div className="text-left">
         <p>
-          Click on an existing <code>button</code> or button-like element to add
-          a button that that button group. You can also select a menu item or
-          nav link.
+          <FontAwesomeIcon icon={faMousePointer} size="lg" /> Click on an
```
Comment (twschiller) [inline src/devTools/editor/panes/insert/InsertMenuItemPane.tsx:57]: 💯

File: src/devTools/editor/panes/insert/InsertPanelPane.tsx [hunk 2]
```diff
@@ -49,9 +51,10 @@ const InsertPanelPane: React.FunctionComponent<{
 
       <div className="text-left">
         <p>
-          Click on a container to insert a panel in that container. Or, click{" "}
-          <span className="text-info">Use Existing Panel</span> to use a panel
-          foundation that already exists for the page
+          <FontAwesomeIcon icon={faMousePointer} size="lg" /> Click on an
```
Comment (twschiller) [inline src/devTools/editor/panes/insert/InsertPanelPane.tsx:54]: 💯

**Review by twschiller — APPROVED** [review]

**Pull request merged**